src/componentLibrary/NewVaultsTable/NewVaultsTable.style.tsx

```diff
@@ -0,0 +1,25 @@
+import styled from 'styled-components';
+import { theme } from 'componentLibrary';
+
+const CoinPairWrapper = styled.div`
```

Comment:
Looking at the design, we also have the currency symbols alongside the icons when the CoinPair component is used in the vault card. Do you think it would make sense to make the currency symbols part of the coin pair component?
<img width="248" alt="Screenshot 2022-06-27 at 13 10 46" src="https://user-images.githubusercontent.com/40243778/175938064-dc807224-56fe-4589-a3e0-725dbe192b40.png">

Reply:
I would rather keep it separated for the sake of simplicity; to avoid need for additional props for text orientation, spacing and also the font size and generally to not constrain the styling. Are you okay with that? :)

Reply:
Yeah totally! This was entirely a question—I don't have a strong opinion on it, and there are pros and cons each way. Very happy for it to stay as it is.